helm/cluster-vsphere/values.yaml

```diff
@@ -231,7 +231,6 @@ global:
       network: {}
       numCPUs: 4
       resourcePool: '*/Resources'
-      template: flatcar-stable-3815.2.2-kube-v1.27.14-gs
```

Comment:
Is this valid to not set it at all?

Reply:
so according to the tests it's working, template is rendered by cluster chart using releases